locations/spiders/ruby_tuesday_us.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+from chompjs import parse_js_object
+from scrapy import Spider
+from scrapy.http import Response
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class RubyTuesdayUsSpider(Spider):
+    name = "ruby_tuesday_us"
+    item_attributes = {"brand": "Ruby Tuesday", "brand_wikidata": "Q7376400", "nsi_id": "rubytuesday-2fabb0"}
```

Comment:
Are `nsi_id` values stable? We have `pipelines/apply_nsi_categories.py` which dynamically finds the relevant NSI entry by matching `brand_wikidata` and `operator_wikidata`, ensuring the latest `nsi_id` is used

Reply:
I think it also blocked the automatic category detection. Thanks for the correction!

Reply:
`nsi_id` isn't stable, and yeah, that will have blocked the automatic categories. Thank you.